bedrock/firefox/templates/firefox/family/includes/modules/faq.html

```diff
@@ -0,0 +1,111 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "firefox/family/includes/macros.html" import module_tag, browser_border %}
+
+{% set csm_utm_params = '?utm_source=firefox&utm_medium=partnership&utm_campaign=faq' %}
+
+<section id="faq" class="c-faq">
+  <div class="mzp-l-content mzp-t-content-xl">
+    {{ module_tag(title='FAQs', anchor='faq') }}
+    <div class="c-card t-shadow">
+      <h3 class="c-subtitle">Not-so FAQs, answered by <span class="c-faq-logo">Common Sense Media</span></h3>
+      <p>Our friends at <a rel="external" href="https://www.commonsensemedia.org/{{ csm_utm_params }}">Common Sense Media</a> lend their expert advice on some of today’s toughest online  parenting scenarios.</p>
+    </div>
+    <h4 class="c-faq-intro">Here are some toughies</h4>
+    <!-- INSTAGRAM -->
+    <details>
```

Comment:
**question for reviewer**: is it better to use the Details component here (with sections)? Not sure the HTML details semantics fit an FAQ section